src/extensibility/registry_utils.js

```diff
@@ -136,8 +136,16 @@ define(function (require, exports, module) {
             sortedEntries.push(registry[key]);
         });
         sortedEntries.sort(function (entry1, entry2) {
-            return getPublishTime((subkey && entry2[subkey]) || entry2) -
-                getPublishTime((subkey && entry1[subkey]) || entry1);
+            if (sortBy !== "publishedDate") {
```

Comment:
Not a big deal, but in general I prefer if you spell out all cases:
```js
if (sortBy === "downloadCount") {
} else if (sortBy === "publishedDate") {
} else {
  console.error("Unknown sortBy " + sortBy ". Defaults to....")
}
```
Or something like this.